src/runtime_src/driver/xclng/tools/awssak/CMakeLists.txt

```diff
@@ -30,6 +32,8 @@ else()
     xrt_awsstatic
     pthread
     rt
+    boost_filesystem
```

Comment:
Will including boost for Amazon be an issue?

Reply:
We now have boost as part of XRT and it was always required for OCL library. So it is okay to include boost for AWS also.